uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/payment/adyen/AdyenPaymentPresenter.kt

```diff
@@ -134,30 +150,37 @@ class AdyenPaymentPresenter(view: PaymentDropInContract.Actions,
     }
 
     private fun createCardConfig(context: Context, publicKey: String): CardConfiguration {
+        val saveCard = !KarhooUISDKConfigurationProvider.isGuest()
+
         return CardConfiguration.Builder(context, publicKey)
-                .setShopperLocale(Locale.getDefault())
-                .setHolderNameRequire(true)
-                .setShowStorePaymentField(false)
-                .build()
+            .setShopperLocale(Locale.getDefault())
+            .setHolderNameRequired(true)
+            .setShowStorePaymentField(saveCard)
```

Comment:
I remember Mostafa set this to false intentionally to fix an inconsistency that was reported. @mionescu95 could you please confirm this was done as part of the hotfix release?

Reply:
Good catch, i'll set it on false